tools/eslint-plugin-minecraft-linting/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "eslint-plugin-minecraft-linting",
-  "version": "1.0.1",
```

Comment:
Don't manually rev this, the change file will do this at publish

Reply:
Ok, gotcha, reverted. I was wondering where all the previous change files went 🤔 .
I saw there are some commits making those updates, thanks.